### ISSUE TITLE
Event Gate linked from a Sliding Rail freezes player after pressing 'F' to enter its config menu

### ISSUE THREAD

**Issue by CurioInventorium** [tsunamayo/Starship-EVO#4571]
### Short Description of the issue:
I linked a Sliding rail to an event gate and pressed F to open the menu to see what I could do. The menu didn't open and the character was frozen in place. I was still able open menus via tab and the main menu, but nothing else.
...

### Steps to reproduce the issue:
Link a sliding rail to an event block and press 'F' while pointed at the event block to open its config menu.
...

### Environment
- Starship EVO version: 22w06d
- Graphics Card: RTX2060

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_
![Starship EVO 12_02_2022 10_48_14](https://user-images.githubusercontent.com/51359034/153708586-0e9ab043-2c42-4ab6-b694-4588987670ea.png)

What was extra frustrating was I was recording a tutorial video and decided to experiment near the end of it, resulting in a bit of an embarrassing 'oh, I've found a bug' moment. That'll teach me to go off script :)

Please also include the `Player.log` file. You can find the log file here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

If you issue concerns a particular design please include any blueprint that could help to reproduce the issue.
Blueprint folder is over at:
`%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints\`
You will need to zip the folder to upload it. 
To do so Right click and choose [Send to -> compressed zip file] then drag the .zip onto your report to upload it.

**Comment by tsunamayo** [comment]
@CurioInventorium Hi, I cant reproduce the issue and you didnt attach a log so I cant investigate ;)
Can you reproduce this all the time? If you can you send me the log (and tell me maybe you are doing something different?)
Thanks!

**Comment by CurioInventorium** [comment]
@tsunamayo It seems to have been fixed since the last time I tested, which was not long after the Space Station update. So I guess this can be closed.
I figured since all that was needed was to link the rail to the event block I didn't need to include the log.

In testing this time I created a fresh platform from a starter block and put down a sliding rail. Then connected that to a query block and the menu opened. I also tested again on the vehicle that it didn't work on before and that worked too.
So all is fine.
